tests/pretty_format_json_test.py

```diff
@@ -26,6 +26,17 @@ def test_unsorted_pretty_format_json(filename, expected_retval):
     assert ret == expected_retval
 
 
+@pytest.mark.parametrize(('filename', 'expected_retval'), (
+        ('not_pretty_formatted_json.json', 1),
+        ('unsorted_pretty_formatted_json.json', 1),
+        ('pretty_formatted_json.json', 1),
+        ('tab_pretty_formatted_json.json', 0),
+))
+def test_unsorted_pretty_format_json(filename, expected_retval):
```

Comment:
(which is why travis-ci was failing, because this name clobbers the other one so the code in the body of the other function was not running, causing coverage to be not-100)